automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -63,12 +79,34 @@ object Leonardo extends RestClient with LazyLogging {
       handleClusterSeqResponse(parseResponse(getRequest(s"$url/$path")))
     }
 
+    def listIncludingDeletedRuntime(
+      googleProject: GoogleProject
+    )(implicit token: AuthToken): Seq[ListRuntimeResponseCopy] = {
+      val path = s"api/google/v1/runtimes/${googleProject.value}?includeDeleted=true"
+      logger.info(s"Listing runtimes including deleted in project: GET /$path")
+      val parsedRequest = parseResponse(getRequest(s"$url/$path"))
+      handleListRuntimeResponse(parseResponse(getRequest(s"$url/$path")))
+    }
+
     def create(googleProject: GoogleProject, clusterName: RuntimeName, clusterRequest: ClusterRequest)(
       implicit token: AuthToken
     ): ClusterCopy = {
+
       val path = clusterPath(googleProject, clusterName, Some(ApiVersion.V2))
       logger.info(s"Create cluster: PUT /$path")
       handleClusterResponse(putRequest(url + path, clusterRequest))
+
+    }
+
+    def createRuntime(googleProject: GoogleProject, clusterName: RuntimeName, clusterRequest: RuntimeRequest)(
```

Comment:
Could we rename `clusterName` to `runtimeName` for the GCE functions?

Reply:
Renamed these!